### ISSUE TITLE
Cannot pull docker Image on linux/arm64

### ISSUE THREAD

**Issue by teejays** [rocket-admin/rocketadmin#742]
**Describe the bug**
Cannot pull docker image.

**To Reproduce**
1. Go to terminal
2. Run `docker pull rocketadmin/rocketadmin`
3. See error
```
$ docker pull rocketadmin/rocketadmin
Using default tag: latest
latest: Pulling from rocketadmin/rocketadmin

What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview rocketadmin/rocketadmin
no matching manifest for linux/arm64/v8 in the manifest list entries
$
```

**Expected behavior**
I expect the image to be pulled successfully. 

**Screenshots**
[NA]

**Desktop (please complete the following information):**
 - OS: MacOS 14.5 (Sonoma)

**Smartphone (please complete the following information):**
 - Device: Macbook Air - M3

**Additional context**
[NA]

**Comment by gugu** [comment]
Right now we need ibm_db npm module, which does not support arm64. We'll think how can we solve this

**Comment by gugu** [comment]
Fixed in version 0.15.1
